CHANGELOG.md

```diff
@@ -9,6 +9,17 @@
 
   Adds `action_policy:install` and `action_policy:policy MODEL` Rails generators.
 
+- Optional authorization target. ([@somenugget][])
```

Comment:
Please, add yourself to the end of the file to make this link work

Reply:
Done 👍🏻